spec/models/comment_spec.rb

```diff
@@ -3,6 +3,17 @@
 describe Comment do
   it_behaves_like "email_confirmable"
 
+  describe "#confirm!" do
+    let(:comment) do
+      VCR.use_cassette('planningalerts') { build(:comment, confirmed: true) }
+    end
+
+    it "should run callback once already confirmed" do
```

Comment:
Yup, there's a typo. @equivalentideas the context makes it way more readable, thanks :ok_hand: